lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/docs/repl.txt

```diff
@@ -17,8 +17,14 @@
 
     Examples
     --------
-    > var y = {{alias}}( TODO, TODO )
-    TODO
+    > var y = {{alias}}( 1, 0.5 )
+    0.5
+    > y = {{alias}}( -1.0, 0.5 )
+	NaN
+	> y = {{alias}}( 0, 0.5 )
```

Comment:
@jrreed83 Looks like tab indentation got inserted both here and on L26. When @Planeshifter finishes the REPL text linter, we'll be able to automatically catch this. Until then, if you have set up EditorConfig in your IDE, then it should read the `.editorconfig` file in the root project directory and automatically set file indentation.